apps/block_scout_web/assets/css/theme/_polygon_hermez_variables.scss

```diff
@@ -0,0 +1,194 @@
+    url("../static/fonts/modern-era/ModernEra-ExtraBold.ttf") format("truetype"),
+    local("Helvetica Neue"), local("Arial");
+  font-weight: 800;
+  font-style: "normal";
+}
+
+$font-family: "Modern Era";
+
+$primary: #8248e5;
+$secondary: #ffc55a;
+$tertiary: #081132;
+
+// hermez global vars
+$hermez-text-color: $tertiary;
+$hermez-text-light-color: #888baa;
+$hermez-text-contrast-color: #b2afc7;
+$hermez-purple: #7b3fe4;
+
+// header
+$hermez-search-box-background: #000414;
+$header-background-color: #081132;
+$header-links-color: $hermez-text-contrast-color;
+$header-links-color-active: #fff;
+$header-icon-color-hover: #fff;
+$header-icon-border-color-hover: #fff;
+
+// dashboard
+$hermez-dashboard-gradient: linear-gradient(180deg, #4e0fb7, #742fe8);
+
+// tiles
+$tile-type-block-color: $hermez-purple;
+$tile-type-transaction-color: #5ea8ff;
+$tile-body-a-color: $hermez-purple;
+
+// footer
+$footer-background-color: $tertiary;
+
+.layout-container {
+  .navbar {
+    .dark-mode-changer {
+      display: none;
+    }
+    .input-group-text {
+      // We need #slash-icon to override the original value, already set as !important
+      &.border#slash-icon {
+        border-color: $hermez-text-contrast-color !important;
+        color: $hermez-text-contrast-color !important;
+        background-color: $hermez-search-box-background !important;
+      }
+    }
+    .search-control {
+      background: $hermez-search-box-background !important;
+      border-color: $hermez-search-box-background !important;
+      &.focused-field {
+        border: $hermez-search-box-background !important;
+      }
+      .main-search-autocomplete {
+        background-color: $hermez-search-box-background !important;
+        &::placeholder {
+          color: $hermez-text-contrast-color;
+        }
+      }
+      .main-search-autocomplete,
+      .main-search-autocomplete-mobile {
+        color: $hermez-text-contrast-color !important;
+      }
+    }
+  }
+  .dashboard-banner-container {
+    background-image: $hermez-dashboard-gradient;
+    .dashboard-banner-network-plain-container {
+      background-color: transparent;
+      box-shadow: none;
+      &::after {
+        background-color: transparent;
+        box-shadow: none;
+      }
+      .dashboard-banner-network-stats {
+        div {
+          &::before {
+            background-color: $secondary;
+          }
+        }
+        .dashboard-banner-network-stats-item {
+          .dashboard-banner-network-stats-label {
+            color: $hermez-text-contrast-color;
+          }
+        }
+      }
+    }
+    .dashboard-banner-network-graph
+      .dashboard-banner-chart-legend
+      .dashboard-banner-chart-legend-item:nth-child(n + 1):nth-child(-n + 2) {
+      // Hides Price and Market Cap info, keeping Tx/day
+      display: none;
+    }
+  }
+  .card {
+    .card-body {
+      .card-title {
+        font-weight: 700;
```

Comment:
We can use the `$font-weight-bold` variable here I think

Reply:
Yep, changed.